src/components/LoginForm/LoginForm.js

```diff
@@ -1,22 +1,56 @@
 import React from 'react';
 
+import './LoginForm.scss';
+import logo from '../../assets/images/Logo.svg';
+import Logo from '../Logo';
+import Title from '../Title';
```

Comment:
Good to have some specific name for this component (e.g., `SomeSpecTitle`, `LoginTitle`)